.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
Please add `Cargo.lock` in this list and remove it from git.

Reply:
I understand why a company would have such a requirement, but I don't see the advantage in this project. Please remove it. We will add it back if we realize that it's needed some some reasons.